### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by nbriannl**
**What is the purpose of this pull request? (put an "X" next to item)**

•  [ ] Documentation update
•  [ ] Bug fix
•  [ ] New feature
•  [ ] Enhancement to an existing feature
• [x] Other, please explain: Improvement of Pull Requests 

Fixes https://github.com/MarkBind/markbind/issues/1214

**What is the rationale for this request?**

It would be nice to have a pull request template for vue-strap like the one we have for markbind

**What changes did you make? (Give an overview)**

Add GitHub pull request template

**Provide some example code that this change will affect:**

\-

**Is there anything you'd like reviewers to focus on?**

It's the same template as MarkBind's, I'm sure we all copy and paste from MarkBind. 
Except for the referencing of links, we have to put the full URL since it's another repo

**Testing instructions:**

\-

### CODE REVIEW

**Comment by le0tan** [comment]
Does this mean we should also have issue tracker for Vue Strap?

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+**What changes did you make? (Give an overview)**
+
+
+**Provide some example code that this change will affect:**
+
+<!-- Paste the example code below: -->
+```js
+
+```
```
Comment (le0tan) [inline .github/PULL_REQUEST_TEMPLATE:34]: Is this section applicable to this repo?

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,52 @@
+**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**
+
+• [ ] Documentation update
```
Comment (le0tan) [inline .github/PULL_REQUEST_TEMPLATE:3]: Not applicable for vue-strap.

**Comment by nbriannl** [comment]
> Does this mean we should also have issue tracker for Vue Strap?

I think there's no need for that. I think it's either to put all issues for both vue strap and markbind in the markbind repo.

File: .github/PULL_REQUEST_TEMPLATE (hunk 1 above)
Comment (openorclose) [reply .github/PULL_REQUEST_TEMPLATE:34]: If there are changes to attributes etc, then this would be applicable i think

**Comment by ang-zeyu** [comment]
Don't think this would be necessary since we are going with https://github.com/MarkBind/markbind/issues/981. Could check with @le0tan on the progress

**Comment by nbriannl** [comment]
Ah, interesting, I can put this on hold since it's a small PR anyways.

**Comment by nbriannl** [comment]
> Don't think this would be necessary since we are going with [MarkBind/markbind#981](https://github.com/MarkBind/markbind/issues/981). Could check with @le0tan on the progress

I completely misread thinking we were on the storybook PR. Are you referring to having an issue tracker for VueStrap, being uneccesary?

File: .github/PULL_REQUEST_TEMPLATE (hunk 1 above)
Comment (nbriannl) [reply .github/PULL_REQUEST_TEMPLATE:33]: Yeah, if anyone were to change anything to the SFCs

**Pull request merged**